docs/content/doc/administration/hardening.en-us.md

```diff
@@ -0,0 +1,70 @@
+---
+date: "2023-07-06T12:30:00+05:00"
+title: "Gitea Hardening"
+slug: "hardening"
+weight: 120
+toc: false
+draft: false
+aliases:
+  - /en-us/hardening
+menu:
+  sidebar:
+    parent: "administration"
+    name: "Gitea hardening"
+    weight: 120
+    identifier: "hardening"
+---
+
+# Gitea hardening
+
+**Table of Contents**
+
+{{< toc >}}
+
+In a multi-user environment, or where Gitea is exposed to the Internet, it may be desirable to harden your installation.
+
+## Block anonymous users
+
+To prevent anonymous users from accessing your Gitea server:
+
+```ini
+[service]
+REQUIRE_SIGNIN_VIEW = true
+```
+
+## Block anonymous registrations
+
+To prevent anonymous users from registering themselves on your Gitea server:
+
+```ini
+[service]
+DISABLE_REGISTRATION     = true
+SHOW_REGISTRATION_BUTTON = false
+```
+
+In such a case, users must be registered manually by a Gitea administrator, or via the Gitea API.
+
+## Harden Gitea server, but allow access to CI/CD server
+
+If Gitea is integrated with a CI/CD server, and both are exposed to the Internet, then additional steps must be taken:
```

Comment:
As gitea has CI/CD as well, we should call this "external CI/CD server" to distinguish.

Reply:
Gitea Actions clones via HTTP/S as well, so maybe something along the lines of "If you are using Gitea with Gitea Actions, or an external CI/CD server..."?

Reply:
But, "If you are using Gitea with Gitea Actions", you don't need to "Create an OAuth2 application for your CI/CD server."

So maybe @silverwind's suggestion is more correct.